packages/venia-concept/src/components/Checkout/paymentsForm.js

```diff
@@ -1,59 +1,45 @@
 import React, { Component } from 'react';
 import { Form } from 'informed';
-import { array, bool, func, shape, string } from 'prop-types';
+import { bool, func, shape, string } from 'prop-types';
 
+import BraintreeDropin from './braintreeDropin';
 import Button from 'src/components/Button';
-import Label from './label';
-import Select from 'src/components/Select';
 
 import classify from 'src/classify';
 import defaultClasses from './paymentsForm.css';
 
 class PaymentsForm extends Component {
     static propTypes = {
-        availablePaymentMethods: array.isRequired,
         cancel: func.isRequired,
         classes: shape({
             body: string,
             footer: string,
             heading: string,
             paymentMethod: string
         }),
-        paymentMethod: string,
         submit: func.isRequired,
         submitting: bool
     };
 
-    static defaultProps = {
-        availablePaymentMethods: [{}]
+    // TODO: don't actually keep the state here, move to Redux
```

Comment:
This TODO may not be accurate.

Either I can move this state to Redux, or I can remove this TODO.